new/LIT6966Bahagara.xml

```diff
@@ -0,0 +1,78 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6966Bahagara" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">በሐገረ፡ ፊልጶስ፡ ቂሳርያ፡ ድኅረ፡ ሰርሮታ፡</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">Ba-ḥagara Filṗos Qisāryā dǝḫra sarǝrotā, ... (Qǝne of za-yǝʾǝze type)</title>
+                <title xml:lang="en" corresp="#t1">In the land of Philipp of Cesarea, after its foundation, ...</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DN"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation notBefore="1580" notAfter="1610"/>
+            <abstract>
+                <p>The poem, attested in <ref type="mss" corresp="BNFabb57"/>, is a qǝne
+                    of the type <foreign xml:lang="gez">za-yǝʾǝze</foreign>, assigned to <persName ref="PRS14051LaekaMaryam"/>.</p>
```

Comment:
term

Reply:
ok

Reply:
Keyword for the type, see https://github.com/BetaMasaheft/Documentation/issues/2504